packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js

```diff
@@ -45,6 +55,15 @@ export default /* @ngInject */ ($stateProvider) => {
           datacenterId,
           productId,
         }),
+      loadLicences: /* @ngInject */ (
```

Comment:
Since this is going to resolve to a `licence` (and not to a method), may be we can rename it to `licence`?